supervisor/snapshots/__init__.py

```diff
@@ -98,8 +98,10 @@ async def import_snapshot(self, tar_file):
 
         # Already exists?
         if snapshot.slug in self.snapshots_obj:
-            _LOGGER.error("Snapshot %s already exists!", snapshot.slug)
-            return None
+            _LOGGER.warning(
+                "Snapshot %s already exists! overwriting snapshot", snapshot.slug
+            )
+            self.remove(self.get(snapshot.slug))
```

Comment:
Since snapshots can't change the content, what is exactly the benefit here?

Reply:
Look at the linked issue.
This allows for re-uploading a snapshot file if the user is impatient, or aborts after initialization.